conf/messages

```diff
@@ -1329,9 +1329,9 @@ itemDensity.change.hidden = density
 itemDensity.suffix = kg/m<sup>3</sup>
 itemDensity.hint = Enter the density, at 15 degrees Celsius per kilogram per cubic metre.
 
-itemsPackagingAddToList.title.singular = You have added 1 packaging type for item {0}
+itemsPackagingAddToList.title.singular = You have added 1 packaging type for item {1}
 itemsPackagingAddToList.title.plural = You have added {0} packaging types for item {1}
-itemsPackagingAddToList.heading.singular = You have added 1 packaging type for item {0}
+itemsPackagingAddToList.heading.singular = You have added 1 packaging type for item {1}
```

Comment:
As above

```suggestion
itemsPackagingAddToList.heading.singular = You have added 1 packaging type for item {0}
```

Reply:
Just ran the code and it runs as intended (showing the correct item number), it does look strange when viewing this solely from a message point of view